utils/apply_cleanup.py

```diff
@@ -0,0 +1,4 @@
+def apply_cleanup(pik_df, functions):
+    for function in functions:
+        pik_df = function(pik_df)
+    return pik_df
```

Comment:
there is no need to create a module for 4 lines of code, but its okay...

Reply:
I find separation of concerns in code structure is always needed, no matter how few code lines are . This function is also expected to get bigger as I add error handling procedures.